### PR TITLE
Match retail text for some actors :heart: 

### PR DESCRIPTION
**Pull request by Dragorn421**
```
src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c
src/overlays/actors/ovl_En_Daiku/z_en_daiku.c
src/overlays/actors/ovl_En_Daiku_Kakariko/z_en_daiku_kakariko.c
src/overlays/actors/ovl_En_Door/z_en_door.c
src/overlays/actors/ovl_En_Holl/z_en_holl.c
```
shoutouts to the `fake_label`

### CODE REVIEW

File: src/overlays/actors/ovl_En_Door/z_en_door.c [hunk 1]
```diff
@@ -195,6 +194,7 @@ void EnDoor_SetupType(EnDoor* this, PlayState* play) {
                 this->actionFunc = EnDoor_AjarWait;
                 this->actor.world.rot.y = -0x1800;
             }
+        fake_label:;
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_En_Door/z_en_door.c:197]: https://decomp.me/scratch/cPdOQ

File: src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c [hunk 2]
```diff
@@ -896,9 +896,7 @@ void DoorShutter_PhantomGanonBarsRaise(DoorShutter* this, PlayState* play) {
     f32 targetOffsetY;
 
     PRINTF("FHG SAKU START !!\n");
-    if (this->isActive != 0) {
-        this->isActive--;
-    }
+    DECR(this->isActive);
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c:899]: https://decomp.me/scratch/xDlhl

also matches with a `if(0){}` in a very specific spot https://decomp.me/scratch/E7oT2

File: src/overlays/actors/ovl_En_Door/z_en_door.c (hunk 1 above)
Comment (cadmic) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:197]: can be matched with a scoped temp: https://decomp.me/scratch/okTFM

File: src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c (hunk 2 above)
Comment (cadmic) [reply src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c:899]: I think this is a reasonable match, DECR helped remove some `if(1) {}`s in EnNiw_Update too

File: src/overlays/actors/ovl_En_Door/z_en_door.c (hunk 1 above)
Comment (Dragorn421) [reply src/overlays/actors/ovl_En_Door/z_en_door.c]: Thanks! made it a player temp

**Review by cadmic — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Pull request merged**